flutterPractice04/main.dart

```diff
@@ -0,0 +1,12 @@
+int sumNumbersInArray(var arr) {
```

Comment:
Please specify the type of arr. Also, please pick a non-abbreviated name. How about `numbers`?